tests/improver-spotdb/03-basic_csv.bats

```diff
@@ -48,8 +49,8 @@
 
   # Run cmp to compare the output and kgo.
   cmp "$TEST_DIR/output.csv" \
-      "$IMPROVER_ACC_TEST_DIR/spotdb/basic_csv/kgo.csv"
+      "$IMPROVER_ACC_TEST_DIR/$KGO"
   [[ "$status" -eq 0 ]]
-  rm "$TEST_DIR/output.csv"
-  rmdir "$TEST_DIR"
+
+  improver_check_recreate_kgo "output.csv" $KGO
 }
```

Comment:
This seems to be missing a new line at the end of the file.

Reply:
✅